aform/src/components/form/ADate.vue

```diff
@@ -1,277 +1,94 @@
 <template>
-	<div
-		v-if="!readonly"
-		:event="event"
-		:colIndex="colIndex"
-		:rowIndex="rowIndex"
-		:tableid="tableid"
-		class="adate"
-		tabindex="0"
-		ref="adatepicker">
-		<table>
-			<tr>
-				<td @click="previousMonth" :tabindex="-1">&lt;</td>
-				<th colspan="5">{{ monthAndYear }}</th>
-				<td @click="nextMonth" :tabindex="-1">&gt;</td>
-			</tr>
-			<tr v-for="rowNo in numberOfRows" :key="rowNo">
-				<!-- TODO: (style) remove inline styling and replace with theme package -->
-				<td
-					v-for="colNo in numberOfColumns"
-					:key="(rowNo - 1) * numberOfColumns + colNo"
-					:contenteditable="false"
-					:spellcheck="false"
-					:tabindex="0"
-					:style="{
-						border: isSelectedDate(currentDates[(rowNo - 1) * numberOfColumns + colNo])
-							? '2px solid var(--focus-cell-outline)'
-							: 'none',
-						borderBottomColor: isTodaysDate(currentDates[(rowNo - 1) * numberOfColumns + colNo])
-							? 'var(--focus-cell-outline)'
-							: 'none',
-					}"
-					@click.prevent.stop="selectDate($event, (rowNo - 1) * numberOfColumns + colNo)"
-					:class="{
-						todaysdate: isTodaysDate(currentDates[(rowNo - 1) * numberOfColumns + colNo]),
-						selecteddate: isSelectedDate(currentDates[(rowNo - 1) * numberOfColumns + colNo]),
-					}">
-					{{ new Date(currentDates[(rowNo - 1) * numberOfColumns + colNo]).getDate() }}
-				</td>
-			</tr>
-		</table>
+	<div>
+		<input
+			ref="dateRef"
+			type="date"
+			:id="uuid"
+			:disabled="readonly"
+			:required="required"
+			:value="inputDate"
```

Comment:
@agritheory I had to move the map from `v-model` to `value` since the two-way sync was causing some issues with the `v-model` setup in the story. Let's discuss this change.